documentation/assemblies/security/assembly-securing-access.adoc

```diff
@@ -7,15 +7,26 @@
 
 [role="_abstract"]
 Secure your Kafka cluster by managing the access a client has to Kafka brokers.
-Specify configuration options to secure Kafka brokers and clients
-
 A secure connection between Kafka brokers and clients can encompass the following:
 
 * Encryption for data exchange
 * Authentication to prove identity
 * Authorization to allow or decline actions executed by users
 
-The authentication and authorization mechanisms specified for a client must match those specified for the Kafka brokers.
+In Strimzi, securing a connection involves configuring listeners and user accounts:
+
+Listener configuration:: Use the `Kafka` resource to configure listeners for client connections to Kafka brokers.
+Listeners define how clients authenticate, such as using TLS, SCRAM-SHA-512, OAuth 2.0, or custom authentication methods.
+To enhance security, configure TLS encryption to secure communication between Kafka brokers and clients.
+You can further secure TLS-based communication by specifying the supported TLS versions and cipher suites in the Kafka broker configuration.
+For an added layer of protection, you can also specify authorization methods in the listener configuration, such as simple, OAuth 2.0, OPA, or custom authorization.
```

Comment:
Kafka supports authorization only at the cluster level. So this is not `in the listener configuration`.

Reply:
I removed the _listener configuration_ bit